roles/network_plugin/calico/tasks/install.yml

```diff
@@ -179,9 +179,10 @@
       },
       "spec": {
           "logSeverityScreen": "Info",
+          {% if not calico_no_global_as_num|default(false) %}"asNumber": {{ global_as_num }},{% endif %}
           "nodeToNodeMeshEnabled": {{ nodeToNodeMeshEnabled|default('true') }} ,
           "serviceExternalIPs": {{ _service_external_ips|default([]) }},
-          "asNumber": {{ global_as_num }} }}
+          "serviceClusterIPs": [{"cidr": {{ kube_service_addresses }} }] }}
```

Comment:
@catblade I forgot this one too...
A condition should be added on the serviceClusterIPs addition (`if calico_advertise_cluster_ips|default(false)` like on the env var in the ds manifest)

Reply:
Should be addressed now.